.env.example

```
@@ -18,6 +18,7 @@ KIMI_API_KEY=                 # kimi.com/code (Kimi 代码计划)
 # Aggregator platforms (optional)
 SILICONFLOW_API_KEY=          # siliconflow.cn
 OPENROUTER_API_KEY=           # openrouter.ai
+ATLASCLOUD_API_KEY=           # atlascloud.ai
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)